content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -571,9 +571,11 @@ If filing a bug, please include detailed information about how to reproduce the
 
 We have a lot of features in our roadmap. An abbreviated high level list is included below, but we encourage you to view our [roadmap project](https://github.com/orgs/kubernetes/projects/8) and help us make Windows support better by [contributing](https://github.com/kubernetes/community/blob/master/sig-windows/).
 
-### CRI-ContainerD
+### CRI-ContainerD (alpha in v1.18)
 
-{{< glossary_tooltip term_id="containerd" >}} is another OCI-compliant runtime that recently graduated as a {{< glossary_tooltip text="CNCF" term_id="cncf" >}} project. It's currently tested on Linux, but 1.3 will bring support for Windows and Hyper-V. [[reference](https://blog.docker.com/2019/02/containerd-graduates-within-the-cncf/)]
+{{< glossary_tooltip term_id="containerd" >}} is another OCI-compliant runtime that recently graduated as a {{< glossary_tooltip text="CNCF" term_id="cncf" >}} project.
+
+Kubernetes v1.18 is tested along with the upcoming ContainerD 1.4 release.
```

Comment:
Please [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future). I would also avoid hinting at the recent past (“recently graduated”). This kind of text tends to go stale.

Reply:
thanks! just fixed